_hackbars/eugli.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: ANY_TITLE
+author: YOUR_NAME
```

Comment:
hey @eugli, here you left something.

you need to add `title` & `author`

Reply:
oops, fixed it